pages/dev-mode.en.mdx

```diff
@@ -1,15 +1,30 @@
 # Dev mode 
 
-We have now added support to Figma's **Dev Mode** for the **Tokens Studio for Figma** plugin.
+We have    added support to Figma's **Dev Mode** for the **Tokens Studio for Figma** plugin.
 
 import Callout from 'nextra-theme-docs/callout'
 
-### How to enable
+### Installation
+**NOTE:** You can only access dev mode for Tokens Studio by toggling on Figma's `Dev mode`. 
+<img src='/figma-dev-mode.png' alt="Toggling On Dev Mode in Figma" width="800"/>
+
+---
+
 When in Figma's dev mode, search for **Tokens Studio for Figma** in the `Plugins` tab, and click on save. 
 In the inspect tab, you can then change it from `CSS` to `Tokens Studio for Figma` under the `Language` dropdown menu. 
 Once selected, you can then proceed to select a frame which will then display the applied tokens in the inspect panel on the right. 
```

Comment:
"Once selected, you can then proceed to select a frame which will then display the applied tokens in the inspect panel on the right."

This is not an installation step. It's part of Using Dev mode